howto/stream/exchange-oob-data.md

```diff
@@ -269,6 +272,55 @@ try {
     ex.printStackTrace();
 }
 ```
+
+#### Install the APK as system app
+
+To connect the data channel to the Anbox WebRTC data proxy service within an Android container, the Android app must be installed and running as a system app. To do so, proceed with the following steps:
+1. Add the attribute `android:sharedUserId="android.uid.system"` into the `<manifest>` tag in the `AndroidManifest.xml` file of your Android app
+2. Create an addon to install your APK as a system app through the pre-start hook with the following content:
+    ```
+    #!/bin/bash -ex
+
+    # Only install the APK as a system app when bootstrapping an application.
+    if  [ "$INSTANCE_TYPE" = "regular" ]; then
+      exit 0
+    fi
+
+    aam install-system-app \
+      --apk="${ADDON_DIR}"/app.apk \
+      --permissions=<comma-separated list of permissions that the application requires> \
+      --package-name=<package_name>
+      --access-hidden-api
+    ```
+
+   See [How to install an APK as a system app](https://documentation.ubuntu.com/anbox-cloud/en/latest/howto/port/install-apk-system-app/#howto-install-apk-system-app) for details.
+
+3. Navigate to the addon directory and add it to AMS:
```

Comment:
If people just blindly follow the above (and they will do) they will fail as the `manifest.yaml` is missing, the hook is maybe not executable etc. If we give steps, they should work and include everything needed to succeed